main.go

```diff
@@ -54,12 +55,29 @@ type HealthCheckWebService struct {
 // }
 
 func (hc *HealthCheckWebService) apiGetIsPrimary(w http.ResponseWriter, r *http.Request) {
+	m := r.URL.Query().Get("max_allowable_byte_lag")
+
+	max_allowable_byte_lag := int64(0)
```

Comment:
is zero an acceptable amount as a default. Like what would happen if I passed it `?max_allowable_byte_log=` or
a negative value
`?max_allowable_byte_log=-1000`

Reply:
Made some changes to the code. If there is no `max_allowable_byte_lag` value provided it will return a 500. currently it is returning a 500 also if `max_allowable_byte_lag` is -1